application-admintools-ui/src/main/resources/AdminTools/CheckSecurityCache.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.1" encoding="UTF-8"?>
+
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+
+<xwikidoc version="1.5" reference="AdminTools.CheckSecurityCache" locale="">
```

Comment:
You have in another PR a similar page (I guess [CheckUserRightsOnPage](https://github.com/xwikisas/application-admintools/pull/126)). Can you apply suggestions from there in here as well? (e.g. `page` id which is too generic, `buttonwrapper` class used, and maybe I'm missing something else)